src/core/runtime/libp2p-browser.js

```diff
@@ -11,14 +12,15 @@ class Node extends libp2p {
   constructor (peerInfo, peerBook, options) {
     options = options || {}
     const wstar = new WebRTCStar()
+    const wsstar = new WebSocketStar({id: peerInfo.id})
 
     const modules = {
-      transport: [new WS(), wstar],
+      transport: [new WS(), wstar, wsstar],
       connection: {
         muxer: [Multiplex],
         crypto: [SECIO]
       },
-      discovery: [wstar.discovery]
+      discovery: [wstar.discovery, wsstar.discovery]
     }
 
     if (options.bootstrap) {
```

Comment:
Following on the comment -- https://github.com/libp2p/js-libp2p/pull/122#issuecomment-347022264 --, this actually should totally work now because websocket-star becomes part of the set of transports supported by default //cc @patrickdet @beanow